robusta_krr/core/models/config.py

```diff
@@ -35,18 +35,18 @@ class Config(pd.BaseSettings):
 
     # Prometheus Settings
     prometheus_url: Optional[str] = pd.Field(None)
-    prometheus_auth_header: Optional[str] = pd.Field(None)
-    prometheus_other_headers: dict[str, str] = pd.Field(default_factory=dict)
+    prometheus_auth_header: Optional[pd.SecretStr] = pd.Field(None)
```

Comment:
I think this will break running it locally, if someone puts a normal string there get_secret_value might throw